ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -223,6 +223,7 @@ descriptor:
   sweeper: "Action performed in a sump to prevent sediment build-up via water turbulence."
   sync: "The simultaneous operation or activity of two or more things."
   tertiary: "Associated with the tertiary (peripheral) loops of a production process."
+  transducer: "A transducer is a device that converts energy from one form to another."
```

Comment:
definition is not quite specific enough. something more like "a device which converts an analog signal into a digital signal, typically used in electromechanical sensoring" would be more appropriate.

Reply:
Yes changed it to transducer_failed_alarm.
Also updated the description